index.js

```diff
@@ -50,107 +53,66 @@ module.exports = {
         ],
       },
     ],
-    '@typescript-eslint/ban-ts-ignore': ['off'], // should be removed as it was replaced with ban-ts-comment (from 3.0.0)
-    '@typescript-eslint/ban-ts-comment': ['off'], // probably, should be warn
-    '@typescript-eslint/dot-notation': ['error'], // +, also, need to set "dot-notation": "off" according to docs
+    '@typescript-eslint/no-explicit-any': 'warn',
```

Comment:
I think it was `error`, no?
I believe we put it to `warn` in core swag app only

Reply:
I set it to warn because a lot of errors across all repos